src/pkg/utils/utils.go

```diff
@@ -430,9 +430,9 @@ func ParseRelease(str string) (string, error) {
 	return release, nil
 }
 
-// PathExists wraps around os.Stat providing a nice interface for checking an existence of a path.
+// PathExists wraps around os.Lstat providing a nice interface for checking an existence of a path.
 func PathExists(path string) bool {
-	if _, err := os.Stat(path); !os.IsNotExist(err) {
+	if _, err := os.Lstat(path); !os.IsNotExist(err) {
```

Comment:
This is why symbolic links are messy. :)

Did you check all our uses of `PathExists` to be sure that it won't introduce some subtle bug?

Reply:
I perused the code and I think the change should be alright. In some cases, the function can be used to check a possible symlink but I think this modification will not affect even those cases.